ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -16,15 +16,23 @@ Demo package contains simple demo to get and visualize result of model inference
 
 ## Prerequisites
 * Python 3.8+
```

Comment:
probably we should limit python version by 3.8 only, as I can see from OpenVINO docs, they claim 3.6-3.8 support on Windows and Mac OS (https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_windows.html and https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_macos.html), and do not specify exact version for Linux explicitly, but from PyPi we can know that it also supports 3.6-3.8 only, inspite the fact that 3.9 python bindings distributed with openvino IRC package.

cc @slyubimt @akladiev @andrew-zaytsev @moslex

Reply:
python 3.9 is included in support for 2022.1

Reply:
that's nice, but I see python3.9 folder in OpenVINO 2021.4.2 python bindings (on windows)

Reply:
limited by 3.8